sympy/solvers/solvers.py

```diff
@@ -980,7 +981,7 @@ def _sympified_list(w):
             f[i] = fi
 
         if isinstance(fi, (bool, BooleanAtom)) or fi.is_Relational:
-            return reduce_inequalities(f, symbols=symbols)
+            return reduce_inequalities(f, symbols=symbols, **flags)
```

Comment:
`reduce_inequalities` can't accept all of the flags pass to solve so this should be `dict=` as well